Support for Monitoring and Tracing (#85)
microsoft/carbon-aware-sdk: GettingStarted.md

This project is using standard [Microsoft.Extensions.Logging](https://docs.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line).  To configure different log levels, please see the documentation at this link.

### Tracing and Monitoring Configuration

> Application monitoring and tracing can be configured by first specifying the Telemetry Provider int eh environment variable. 

Reviewer: ```suggestion
Application monitoring and tracing can be configured using the `TELEMETRY_PROVIDER` variable in the application configuration. 
```

```bash
TELEMETRY_PROVIDER= "ApplicationInsights"
```

> This application is integrated with Application Insights for monitoring purposes. The telemetry collected in the app is pushed to AppInsights and can be tracked for logs, exceptions, traces and more. To connect to your Application Insights instance, set the following as an environment variable:

Reviewer: ```suggestion
This application is integrated with Application Insights for monitoring purposes. The telemetry collected in the app is pushed to AppInsights and can be tracked for logs, exceptions, traces and more. To connect to your Application Insights instance, configure the `APPLICATIONINSIGHTS_CONNECTION_STRING` variable:
```
